src/loraconf.sample.h

```diff
@@ -14,7 +14,15 @@
 // Note: Use same format as in TTN console (cut & paste, for your convenience)
 // *** Take care : If Using a board with Microchip 24AA02E64 Uinique ID for deveui, **
 // *** this DEVEUI will be overwriten by the one contained in the Microchip module ***
+#if defined (ttgov2)

```

Comment:
what is the purpose of these board specific DEVEUI pref-definitions?

Reply:
It's because I've got different DEVEUI for each board, and as I'm testing on 3 different boards each time, it avoid me the target board using already used deveui when compiling testing. And when I forget to change deveui on each compilation
See loraconf.sample.h you'll see better

Reply:
ok, but for this purpose is loraconf.h (which is kept private by .gitignore), not the template loraconf.sample.h